src/app/views/login/login.component.ts

```diff
@@ -23,7 +23,7 @@ export class LoginComponent {
       .then(result => this.router.navigate(['/home']))
       .catch(errors => {
         if (errors.details) {
-          this.errorMessage = errors.details.password;
+          this.errorMessage = errors.details.password.charAt(0).toUpperCase() + errors.details.password.slice(1);
```

Comment:
Are the error messages capitalised in the designs?

How well this will work in other languages arabic etc, I don't think arabic has capital letters, potentially `toUpperCase` wont make any difference then.